google-cloud-firestore/src/main/java/com/google/cloud/firestore/QueryDocumentSnapshot.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
I know this is a pain, but could you update the copyright line? We updated it across the codebase. It should be:

```
Copyright 2018 Google LLC
```

Reply:
No problem, done.